### ISSUE TITLE
Autokeras not able to load best model

### ISSUE THREAD

**Issue by PhilippeCodes** [keras-team/autokeras#964]
### Bug Description

I was spending the whole day trying to play around with autokeras for an image classification problem. I installed the newest version and updated tensorflow to 2.1.0. I never got it to work with my (Kaggle leaf) dataset so I figgured that I would just try it with the example code from the autokeras website:

```
import autokeras as ak

(x_train, y_train), (x_test, y_test) = mnist.load_data()
print(x_train.shape)  # (60000, 28, 28)
print(y_train.shape)  # (60000,)
print(y_train[:3])  # array([7, 2, 1], dtype=uint8)


clf = ak.ImageClassifier(max_trials=3)

clf.fit(x_train, y_train)

print('Accuracy: {accuracy}'.format(
    accuracy=clf.evaluate(x_test, y_test)))
```

The whole thing seems to do just fine during training of different models having an accuracy or >0.9. However, in the end it says: "INFO:tensorflow:Oracle triggered exit" and the finished model has an accuracy that is no better than random guesses. Furthermore, I get the following warnings:
```
INFO:tensorflow:Oracle triggered exit
Train for 1875 steps, validate for 375 steps
Epoch 1/2
1875/1875 [==============================] - 85s 45ms/step - loss: 2.3015 - accuracy: 0.1122 - val_loss: 2.3016 - val_accuracy: 0.1060
Epoch 2/2
1875/1875 [==============================] - 78s 42ms/step - loss: 2.3013 - accuracy: 0.1124 - val_loss: 2.3016 - val_accuracy: 0.1060
313/313 [==============================] - 5s 15ms/step - loss: 2.3012 - accuracy: 0.1135
Accuracy: [2.3011629931842936, 0.1135]
WARNING:tensorflow:Unresolved object in checkpoint: (root).optimizer.iter
WARNING:tensorflow:Unresolved object in checkpoint: (root).optimizer.beta_1
WARNING:tensorflow:Unresolved object in checkpoint: (root).optimizer.beta_2
WARNING:tensorflow:Unresolved object in checkpoint: (root).optimizer.decay
WARNING:tensorflow:Unresolved object in checkpoint: (root).optimizer.learning_rate
```
It seems like autokeras is unable to retrieve its best models. Can anyone help me with this issue?

**Comment by justinmulli** [comment]
Did you figure this out? I have the same issue. I let it run overnight and the results are useless, though it worked the first time. Lost a lot of time

**Comment by oscaralvaro** [comment]
has this bug being fixed yet?
